readme.md

```diff
@@ -1,47 +1,47 @@
 ## What is in this repository?
-Here are complete set of design files, manufacturing data, source files, test results and other resources that I used the process of building nanovoltmeter. This process was not a coincidence, but response to nanovoltmeter challenge by TiN on [his website](https://xdevs.com/article/nvm_comp/)
+Here are the complete set of design files, manufacturing data, source files, test results and other resources that I used in the process of building a nanovoltmeter. This process was not a coincidence, but a response to the nanovoltmeter challenge by TiN on [his website](https://xdevs.com/article/nvm_comp/)
```

Comment:
"Here is the" is the correct use, as "complete set" is singular. "Here are the files" would also be correct.

Reply:
Done, changes made.